### PR TITLE
Feature/incident forecast

### PR DESCRIPTION
**Pull request by kevgliss**
Initial work on using fbprophet for incident forecasting.

### CODE REVIEW

File: src/dispatch/incident/metrics.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    for (last_day, items) in groupby(incidents_sorted, grouper):
+        dataframe_dict["ds"].append(last_day)
+        dataframe_dict["y"].append(len(list(items)))
+
+    dataframe = pd.DataFrame.from_dict(dataframe_dict)
+
+    forecaster = Prophet()
+    forecaster.fit(dataframe, algorithm="LBFGS")
+
+    future = forecaster.make_future_dataframe(periods=2, freq="M")
```
Comment (snkilmartin) [inline src/dispatch/incident/metrics.py]: I'm currently using 24 for periods

File: src/dispatch/incident/metrics.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    forecast = forecaster.predict(future)
+
+    return forecast
```
Comment (snkilmartin) [inline src/dispatch/incident/metrics.py]: The [Prophet docs](https://facebook.github.io/prophet/docs/quick_start.html#python-api) are a great reference; this line from the Prophet docs might be good to include in the Dispatch docs:

> The forecast object here is a new dataframe that includes a column yhat with the forecast, as well as columns for components and uncertainty intervals.

Comment (kevgliss) [reply src/dispatch/incident/metrics.py]: Thank you! I was going to ask what all of the output values mean.

File: src/dispatch/incident/metrics.py (hunk 1 above)
Comment (kevgliss) [reply src/dispatch/incident/metrics.py]: Yes, I will make this configurable and default to 24.

**Pull request merged**